### DISCUSSION TITLE
可不可以分享一下bili接口文档？

### DISCUSSION THREAD

**Discussion by narcissusxu** [Richasy/Bili.Uwp#278] [问答]
想用electron重新做一个全平台的客户端，但是不知道在哪儿查文档

**Comment by Q-Tai-mu** [comment]
兄弟，要行动起来！！！
![lili](https://user-images.githubusercontent.com/58099119/134774603-ba08a8b7-6117-442c-a98a-f48b7e185cc3.png)

**Comment by hez2010** [comment]
然而用 electron 的话就和 chromium 一样耗电、占内存、启动速度慢，并且安装体积还更大，失去了客户端的意义，这样的话还不如直接去用网页版

**Reply by Q-Tai-mu** [reply]
世界并无完美之物，你要是鸡蛋里挑骨头，那就当我没说过

**Reply by hez2010** [reply]
> 世界并无完美之物，你要是鸡蛋里挑骨头，那就当我没说过

这并不是鸡蛋里挑骨头，做客户端的意义就在于提供给用户比网页版更好的用户体验（更快的加载速度、更省的资源占用、更丝滑的过渡动画......），否则就没有必要做客户端了，变成了为了客户端而客户端
